### PR TITLE
Group overview page

### PR DESCRIPTION
**Pull request by ekes**
Starter for something for localgovdrupal/localgov_microsites#77

### CODE REVIEW

**Comment by ekes** [comment]
Requires config/optional install for the entity view, and view changes.

File: config/optional/views.view.localgov_microsites_overview.yml [hunk 1]
```diff
@@ -1,11 +1,16 @@
+uuid: 109e7ad1-aa4e-40f5-b23d-c2dd0fb12bfb
```
Comment (finnlewis) [inline config/optional/views.view.localgov_microsites_overview.yml]: Does this break things @ekes ?

File: config/optional/views.view.localgov_microsites_overview.yml [hunk 2]
```diff
@@ -1,11 +1,16 @@
...
 dependencies:
   config:
+    - domain.record.localgov_micro_lndo_site
     - group.type.microsite
     - system.menu.admin
   module:
+    - domain
     - group
+_core:
+  default_config_hash: Ap0g2LySVHQfQktUhNR5sbIeLDUNecT1rLgAu2rjb1c
```
Comment (finnlewis) [inline config/optional/views.view.localgov_microsites_overview.yml]: And these lines, do we need to remove them? I think on my install the view has not been created.

File: config/optional/views.view.localgov_microsites_overview.yml [hunk 3]
```diff
@@ -167,8 +172,10 @@ display:
           sort_asc_label: Asc
           sort_desc_label: Desc
       access:
-        type: none
-        options: {  }
+        type: domain
```
Comment (finnlewis) [inline config/optional/views.view.localgov_microsites_overview.yml]: Although, perhaps it is new access control?

File: config/optional/views.view.localgov_microsites_overview.yml (hunk 1 above)
Comment (ekes) [inline config/optional/views.view.localgov_microsites_overview.yml]: UUID needs to go

File: config/optional/views.view.localgov_microsites_overview.yml [hunk 4]
```diff
@@ -1,11 +1,16 @@
+uuid: 109e7ad1-aa4e-40f5-b23d-c2dd0fb12bfb
 langcode: en
 status: true
 dependencies:
   config:
+    - domain.record.localgov_micro_lndo_site
```
Comment (ekes) [inline config/optional/views.view.localgov_microsites_overview.yml]: Don't want this dependency? Where from

File: config/optional/views.view.localgov_microsites_overview.yml [hunk 5]
```diff
@@ -1,11 +1,16 @@
...
     - group.type.microsite
     - system.menu.admin
   module:
+    - domain
     - group
+_core:
```
Comment (ekes) [inline config/optional/views.view.localgov_microsites_overview.yml]: And this.

**Review by finnlewis — APPROVED** [review]
Nice @ekes !!  ! J'approuve!

File: config/optional/views.view.localgov_microsites_overview.yml (hunk 3 above)
Comment (ekes) [reply config/optional/views.view.localgov_microsites_overview.yml]: These should all not be in the PR any more.

The current branch shouldn't have a change on `config/optional/views.view.localgov_microsites_overview.yml`
So shouldn't touch the microsites admin page.

File: config/optional/views.view.localgov_microsites_overview.yml (hunk 2 above)
Comment (ekes) [reply config/optional/views.view.localgov_microsites_overview.yml]: File removed from the branch with [4771945](https://github.com/localgovdrupal/localgov_microsites_group/pull/44/commits/4771945adf3acfc17c9bf8c9ef44f1c2e7524442)

**Pull request merged**